### PR TITLE
Feature/workouts with tests

### PR DESCRIPTION
**Pull request by kotleta2007**

### CODE REVIEW

File: app/src/androidTest/java/com/sdp/movemeet/workout/WorkoutTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    public void TextWorkoutActivityIsCorrect() {
+        try (ActivityScenario<TextWorkoutActivity> scenario = ActivityScenario.launch(TextWorkoutActivity.class)) {
+
+            try {
+                Thread.sleep(1000);
+            } catch (InterruptedException e) {
+                assert (false);
+            }
+
+            //onView(withId(R.id.textView)).check(matches(withText(TEST_NAME)));
```
Comment (jlstoupy98) [inline app/src/androidTest/java/com/sdp/movemeet/workout/WorkoutTest.java:59]: why this check is commented ?

File: app/src/main/java/com/sdp/movemeet/ui/workout/ListViewAdapter.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
+package com.sdp.movemeet.ui.workout;
+
```
Comment (RoxaneBurri) [inline app/src/main/java/com/sdp/movemeet/ui/workout/ListViewAdapter.java:2]: What is the goal of a list view adapter ?

File: app/src/main/java/com/sdp/movemeet/ui/workout/PageViewModel.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                case 1:
+                    return "Text-based descriptions of workouts";
+                case 2:
+                    return "Images of workouts";
+                case 3:
+                    return "Workouts in PDF format";
+            }
+        }
+    });
+     */
```
Comment (jlstoupy98) [inline app/src/main/java/com/sdp/movemeet/ui/workout/PageViewModel.java:43]: You can delete this unused part

File: app/src/main/java/com/sdp/movemeet/ui/workout/Model.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package com.sdp.movemeet.ui.workout;
+
+public class Model {
```
Comment (RoxaneBurri) [inline app/src/main/java/com/sdp/movemeet/ui/workout/Model.java:3]: This is the model for the workout ?

File: app/src/main/java/com/sdp/movemeet/ui/workout/SectionsPagerAdapter.java [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+import android.content.Context;
+
+import androidx.annotation.Nullable;
+import androidx.annotation.StringRes;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentPagerAdapter;
+
+import com.sdp.movemeet.R;
```
Comment (RoxaneBurri) [inline app/src/main/java/com/sdp/movemeet/ui/workout/SectionsPagerAdapter.java:11]: What is the section pager adapter ?

File: app/src/main/java/com/sdp/movemeet/ui/workout/Model.java (hunk 4 above)
Comment (kotleta2007) [reply app/src/main/java/com/sdp/movemeet/ui/workout/Model.java:3]: Yes, exactly! It describes a Workout with a name, description and icon.

File: app/src/main/java/com/sdp/movemeet/ui/workout/ListViewAdapter.java (hunk 2 above)
Comment (kotleta2007) [reply app/src/main/java/com/sdp/movemeet/ui/workout/ListViewAdapter.java:2]: It basically takes care of displaying the list of available workouts.

File: app/src/main/java/com/sdp/movemeet/ui/workout/PageViewModel.java (hunk 3 above)
Comment (kotleta2007) [reply app/src/main/java/com/sdp/movemeet/ui/workout/PageViewModel.java:43]: You are correct! In my next PR, I will get rid of this 'dead code'.

File: app/src/androidTest/java/com/sdp/movemeet/workout/WorkoutTest.java (hunk 1 above)
Comment (kotleta2007) [reply app/src/androidTest/java/com/sdp/movemeet/workout/WorkoutTest.java:59]: I will work on enriching these tests by checking the content of text fields.

File: app/src/main/java/com/sdp/movemeet/ui/workout/SectionsPagerAdapter.java (hunk 5 above)
Comment (kotleta2007) [reply app/src/main/java/com/sdp/movemeet/ui/workout/SectionsPagerAdapter.java:11]: It organizes the workouts into 'tabs' which you can see on the top of the workout screen.

**Review by RoxaneBurri — APPROVED** [review]

**Review by jlstoupy98 — APPROVED** [review]

**Pull request merged**